packages/expo-cli/src/commands/upload.js

```diff
@@ -33,6 +33,7 @@ export default program => {
     'sku',
     'language',
     'publicUrl',
+    'changelog'
```

Comment:
I believe that for the `changelog` to be passed correctly to the App Store API, you'll also need to add it to to the [options of the upload command](https://github.com/expo/expo-cli/blob/a741eb76fdd1f62eb6194bec97c9c3135be12c3c/packages/expo-cli/src/commands/upload.js#L41-L69)
and [IOSUploader.js](https://github.com/expo/expo-cli/blob/a741eb76fdd1f62eb6194bec97c9c3135be12c3c/packages/expo-cli/src/commands/upload/IOSUploader.js#L174-L188). Please test the command to make sure it works.